### PR TITLE
Disable zone commands

### PR DESCRIPTION
**Pull request by darionyaphet**
#### What type of PR is this?
- [ ] bug
- [ ] feature
- [ ] enhancement

#### What problem(s) does this PR solve?
Issue(s) number: 

Description:

#### Checklist:
Tests:
- [ ] Unit test(positive and negative cases)
- [ ] Function test
- [ ] Performance test
- [ ] N/A

Affects:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

#### Release notes:

Please confirm whether to be reflected in release notes and how to describe:
> ex. Fixed the bug .....

### CODE REVIEW

**Review by critical27 — APPROVED** [review]

**Review by panda-sheep — APPROVED** [review]

File: tests/tck/features/schema/Comment.feature [hunk 1]
```diff
@@ -69,20 +69,20 @@ Feature: Schema Comment
...
-    When executing query:
-      """
-      DESC SPACE test_comment_empty;
-      """
-    Then the result should be, in any order:
-      | ID    | Name                 | Partition Number | Replica Factor | Charset | Collate    | Vid Type          | Atomic Edge | Zones          | Comment |
-      | /\d+/ | "test_comment_empty" | 100              | 1              | "utf8"  | "utf8_bin" | "FIXED_STRING(8)" | false       | "default_zone" | ""      |
+    # When try to execute query:
+    # """
+    # SHOW CREATE SPACE test_comment_empty;
```
Comment (CPWstatic) [inline tests/tck/features/schema/Comment.feature:74]: Why disable this test? We should fix it.

Comment (darionyaphet) [reply tests/tck/features/schema/Comment.feature:74]: because the zone name is automatically generated by address we can't control

Comment (CPWstatic) [reply tests/tck/features/schema/Comment.feature:74]: We could use regexp

Comment (darionyaphet) [reply tests/tck/features/schema/Comment.feature]: fixed and please go on ~

**Review by critical27 — APPROVED** [review]

**Review by CPWstatic — APPROVED** [review]

**Review by critical27 — APPROVED** [review]

**Pull request merged**